dehydrated

```diff
@@ -126,7 +126,7 @@ jsonsh() {
     fi
 
     # shellcheck disable=SC2196
```

Comment:
The shellcheck comment/command should be removed as it is not needed anymore.

Reply:
Thanks, done.

Reply:
That was fast. :smile: